demo/services/mockWorkspaceWorkerPoolQueuesApi.ts

```diff
@@ -49,8 +49,7 @@ export class MockWorkspaceWorkerPoolQueuesApi extends MockApi implements IWorksp
   public async deleteWorkerPoolQueue(workerPoolName: string, queueName: string): Promise<void> {
     const workerPool = this.workerPools.find(workerPool => workerPool.name === workerPoolName)!
     const workerPoolQueue = this.workerPoolQueues.find(workerPoolQueue => workerPoolQueue.name === queueName && workerPoolQueue.workerPoolId === workerPool.id)!
-
-    return await this.workerPoolQueues.delete(workerPoolQueue.name)
+    return await this.workerPoolQueues.delete(workerPoolQueue.id)
```

Comment:
This ensures the delete is actually registered by the data service; even though our API deletes by name, the data service uses ids